registry/src/pages/admin.js

```diff
@@ -0,0 +1,376 @@
+import React, { useEffect, useState } from "react";
+import { Container } from "react-bootstrap";
+import {
+  MDBBtn,
+  MDBModal,
+  MDBModalDialog,
+  MDBModalContent,
+  MDBModalHeader,
+  MDBModalTitle,
+  MDBModalBody,
+  MDBModalFooter,
+  MDBIcon,
+} from "mdb-react-ui-kit";
+
+const AdminSection = () => {
+  const [deprecateModal, setdeprecateModal] = useState(false);
+  const [deleteNamespaceModal, setdeleteNamespaceModal] = useState(false);
+  const [deletePackageModal, setdeletePackageModal] = useState(false);
+  const [deleteUserModal, setdeleteUserModal] = useState(false);
+  const [deleteReleaseModal, setdeleteReleaseModal] = useState(false);
+  const [changePasswordModal, setchangePasswordModal] = useState(false);
+  const toggleShowDeprecateModal = () => setdeprecateModal(!deprecateModal);
+  const toggleShowDeleteUserModal = () => setdeleteUserModal(!deleteUserModal);
+  const toggleShowDeleteNamespaceModal = () =>
+    setdeleteNamespaceModal(!deleteNamespaceModal);
+  const toggleShowDeletePackageModal = () =>
+    setdeletePackageModal(!deletePackageModal);
+  const toggleShowDeleteReleaseModal = () =>
+    setdeleteReleaseModal(!deleteReleaseModal);
+  const toggleShowChangePasswordModal = () =>
+    setchangePasswordModal(!changePasswordModal);
+  const [packageName, setPackageName] = useState("");
+  const [deprecatepackageName, setdeprecatepackageName] = useState("");
+  const [toggleSwitch, setToggleSwitch] = useState(false);
+  const [namespaceName, setNamespaceName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [releaseName, setReleaseName] = useState("");
+  const [newPassword, setNewPassword] = useState("");
+//   const [isAdmin, setIsAdmin] = 
+
+  useEffect(() => {
+   
+    //   dispatch(resetErrorMessage());
+  
+  }, [isAuthenticated]);
+
+
+  const handleDeprecatePackage = () => {
```

Comment:
This and the below handler can be inside the actions.

Reply:
sure.